README.yaml

```diff
@@ -0,0 +1,90 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-iam-user
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-iam-user
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-iam-user.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-iam-user"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-iam-user.svg"
+    url: "https://github.com/cloudposse/terraform-aws-iam-user/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
```

Comment:
add https://github.com/cloudposse/terraform-aws-iam-assumed-roles to related since it's related and is used in the example